test/service/test_response.py

```diff
@@ -1261,6 +1270,230 @@ def test_bundles_outer_entity(self):
         self.assertEqual(len(response['hits']), len(hits_uuids))
         self.assertSetEqual(indexed_uuids, hits_uuids)
 
+    expected_response = []
+
+    def test_ranged_values(self):
+        test_hits = [{'cellLines': [{'cellLineType': ['induced pluripotent'],
```

Comment:
The formatting of this data structure seems a bit random, maybe dump it into an online formatter?